README.md

```diff
@@ -30,6 +30,13 @@ Yep. That's it. You should see the following output:
 
 ```
 
+```
+## Build
+
+Run "sudo rm -r <Directory_name_of_the_reporisoty>" to correctly compile the project
```

Comment:
please spell this correctly.  perhaps this build-step is better-lived in the docker file?